ES/ESrqg015.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="ESrqg015" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Psalter</title>
+            <editor role="cataloguer" key="SA"/>
+            <editor key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:47:04.296+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0248RQG"/>
+                  <collection>Ethio-SPaRe</collection>
+                  <idno>RQG-015</idno>
+               </msIdentifier>
+               <msContents>
+                  <summary>I) Dāwit “Psalter” (fols. 1r-186vb); I-1) Mazmura Dāwit “Psalms of
+                     David” (fols. 1r-145v); I-2) Maḥālǝya nabiyāt “The Canticles of the
+                     Prophets” (fols. 146r-161r); I-3) Maḥālǝya maḥālǝy “Song of Songs” (fols.
+                     161r-169v); I-4) Wǝddāse Māryām “Praise of Mary” (fols. 170ra-186vb); Additiones 1-3.</summary>
+                  
+                  <msItem xml:id="ms_i1">
+                     
+                     <title type="complete" ref="LIT6368PsaltComm">The
+                        Ethiopic Psalter with Amharic commentary </title>                     
+                     <note>The commentary stopps at the Song of Songs.</note>
+
```

Comment:
add the subitems at least as a note under msItem 1 to make them visible I-1) Mazmura Dāwit “Psalms of
                     David” (fols. 1r-145v); I-2) Maḥālǝya nabiyāt “The Canticles of the
                     Prophets” (fols. 146r-161r); I-3) Maḥālǝya maḥālǝy “Song of Songs” (fols.
                     161r-169v); I-4) Wǝddāse Māryām “Praise of Mary” (fols. 170ra-186vb);